packages/api/internal/orchestrator/orchestrator.go

```diff
@@ -39,15 +41,20 @@ func New(
 		logger.Errorf("Error initializing Analytics client\n: %v", err)
 	}
 
+	port := utils.RequiredEnv("DNS_PORT", "Local DNS server resolving IPs for sandboxes")
 	dnsServer := dns.New()
+	intPort, convErr := strconv.Atoi(port)
```

Comment:
in the start function (which takes an `int`) [here](https://github.com/e2b-dev/infra/blob/main/packages/api/internal/dns/server.go#L87-L100) we just convert it _back_ to string (and if it's not a valid port number it'll error there.

Admittedly the Start method doesn't return in the foreground which means we either have to panic, or continue without a functioning DNS server (this has implications for #261). 

While we're (I'm) being pedantic, a valid `int` isn't neccessarily a valid port number: port numbers have to be valid `uint16` numbers and `int` is basically always an `int64` (but sometimes an `int32` though in practice we may be confident that we'll never need to run this software from a 32-bit build). So we could pass this _and still_ have an invalid port number.